data_quality/config/data_quality/checkpoint.yaml

```diff
@@ -0,0 +1,7 @@
+container_name: staging
```

Comment:
I'm not sure about file name checkpoint.yaml. Maybe expectations.yaml?

Reply:
This is only example config, @lorraine-houston-ensono will be updating it going forward.

Reply:
ok, fine by me

Reply:
So GX has changed to only support running validationg through checkpoints not just running validator.validate() so we're going to have t have a checkpoint file and a config file now. As Satender said, this is only a starting point to prove the deployment works